# Complex test array, with submessages collapsed into columns

std_msgs/Header header

uint16[] x
uint16[] y
uint64[] ts
bool[] polarity
